tests/e2e_all/dockerfiles/daemon/Dockerfile.build.branch.dart.noports

```diff
@@ -0,0 +1,31 @@
+FROM dart:3.6.0
```

Comment:
Should be 3.7.0 and pinned to a hash

Reply:
Where can I find instructions on how to do that again?

Reply:
Never mind , figured it out :)